client/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+    <head>
+        <title>Client Served</title>
+        <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+        <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+    </head>
+    <body>
+        <div id='carousel'>Server has responded with index!</div>
```

Comment:
This text doesn't show up on the page?

Reply:
it does, but it gets replaced.  i'll replace index.html tonight